src/i18n/strings/en_EN.json

```diff
@@ -1579,6 +1579,11 @@
     "Privacy": "Privacy",
     "Share anonymous data to help us identify issues. Nothing personal. No third parties.": "Share anonymous data to help us identify issues. Nothing personal. No third parties.",
     "Sessions": "Sessions",
+    "You can use this device to sign in a new device with a QR code. ": "You can use this device to sign in a new device with a QR code. ",
```

Comment:
This has an extra space at the end of it that shouldn't be there :)

Reply:
it's been mentioned that these spaces might be intentional: this is a sign that the wrong amount of the string is under the translations. We can't join strings in-code because some languages just don't work like that. Instead, we put entire paragraphs under a single string (repeating some copy if required).